doc/releases/release-notes-2.3.rst

```diff
@@ -153,54 +153,71 @@ Boards & SoC Support
 
 .. rst-class:: rst-columns
 
-   * <TBD>
+   * STM32L5 series of Ultra-low-power MCUs
 
 * Added support for these ARM boards:
 
   .. rst-class:: rst-columns
 
-     * <TBD>
+     * 96Boards AeroCore 2
+     * Adafruit Feather STM32F405 Express
+     * Black STM32 F407VE Development Board
+     * Black STM32 F407ZG Pro Development Board
+     * ST Nucleo H743ZI
+     * ST Nucleo F303RE
+     * ST Nucleo L552ZE-Q
 
 * Added support for these following shields:
 
   .. rst-class:: rst-columns
 
-     * <TBD>
+     * Espressif ESP-8266 Module
+     * MikroElektronika ADC Click
+     * MikroElectronica Eth Click
+     * ST X-NUCLEO-IKS02A1: MEMS Inertial and Environmental Multi sensor shield
 
 Drivers and Sensors
 *******************
 
 * ADC
 
-  * <TBD>
+  * Add support for STM32G4, STM32L1 and STM32H7 series
+  * Enable internal voltage reference source on stm32
```

Comment:
Internal voltage is not reported (`adc_ref_internal` will always return 0), so not sure if this claim is accurate.

Reply:
Even if it's not reported, can it be used as a reference internally?

Reply:
This is coming from https://github.com/zephyrproject-rtos/zephyr/pull/24292

Reply:
Thanks @erwango. As detailed in #24292 it seems that the API would need some small changes to represent the STM32 reference system, so it can be explored in the future.